src/test/java/com/xceptance/neodymium/tests/FixedRandomnessOfDataSetsTests.java

```diff
@@ -0,0 +1,41 @@
+package com.xceptance.neodymium.tests;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.aeonbits.owner.ConfigFactory;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.JUnitCore;
+import org.junit.runner.Result;
+
+import com.xceptance.neodymium.testclasses.data.FixedRandomnessOfDataSets;
+import com.xceptance.neodymium.util.Neodymium;
+
+public class FixedRandomnessOfDataSetsTests extends NeodymiumTest
+{
+    @BeforeClass
+    public static void setUpNeodymiumConfiguration() throws IOException
+    {
+        // set up a temp-neodymium.properties
+        final String fileLocation = "config/temp-neodymium.properties";
+        File tempConfigFile = new File("./" + fileLocation);
+        tempFiles.add(tempConfigFile);
+        Map<String, String> properties = new HashMap<>();
+
+        properties.put("neodymium.context.random.initialValue", "1323");
+
+        writeMapToPropertiesFile(properties, tempConfigFile);
+        ConfigFactory.setProperty(Neodymium.TEMPORARY_CONFIG_FILE_PROPERTY_NAME, "file:" + fileLocation);
+    }
+
+    @Test
+    public void testFixedRandomnessOfDataSets()
+    {
+        // test fixed random data sets support
+        Result result = JUnitCore.runClasses(FixedRandomnessOfDataSets.class);
+        checkPass(result, 5, 0);
+    }
+}
```

Comment:
Please add a newline add the end of every new file.

Reply:
Done